packages/core/src/types/index.ts

```diff
@@ -7,7 +7,8 @@ import {
 
 export type Chain = {
   id: number
-  name: AddEthereumChainParameter['chainName']
+  name: string
+  displayName: AddEthereumChainParameter['chainName']
```

Comment:
I'm good with this, since it [aligns with ethers](https://github.com/ethers-io/ethers.js/blob/948f77050dae884fe88932fd88af75560aac9d78/packages/networks/src.ts/index.ts#L132=), but it is kind of a small breaking change. Maybe we can keep `name` as the "display" name and find a better term for the ethers network name?

Reply:
A comment saying what the difference is between the "display" name and ethers network name would be good too.